src/lib/scheduler_clients/pbs/models.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) 2025, ETH Zurich. All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from typing import Dict, Optional
+
+from pydantic import AliasChoices, Field, validator
+
+# models
+from lib.models.base_model import CamelModel
+from lib.scheduler_clients.models import (
+    JobMetadataModel,
+    JobModel,
+    JobDescriptionModel,
+    JobTime,
+    NodeModel,
+    PartitionModel,
+    ReservationModel,
+)
+
+from datetime import datetime
+
+
+class PbsJobDescription(JobDescriptionModel):
```

Comment:
As mentioned in /firecrest/compute/models.py we should have a "unified" model defined in JobDescriptionModel and here implement the necessary mapping from the unified model to the Pbs one.

For example we can add an alias to map:

- partition -> queue
- constraints -> resources